HTML-CSS/Tasarım ödevi/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+    background-color: rgb(61, 61, 61);
+}
+
+.grid {
+    display: grid; 
+ 
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
Grid kullanman çok iyi bir seçim, layout (düzen) oluştururken genel olarak CSS Grid tercih edilmektedir. Kişisel bir öneri yapmak gerekirse, kodunda gördüğüm kadarıyla aynı şeyi (1fr) üst üste 4-5 defa yazmışsın. Bu her ne kadar çalışsa da pek profesyonel değil, burada kullanabileceğin **repeat** fonksiyon mevcut. Bir göz atmanı öneririm.

CSS Repeat Fonksiyonu MDN Makalesi: https://developer.mozilla.org/en-US/docs/Web/CSS/repeat